lib/msal-common/src/utils/Constants.ts

```diff
@@ -401,3 +401,38 @@ export enum JsonTypes {
     Jwt = "JWT",
     Jwk = "JWK"
 }
+
+export enum SilentTokenRetrievalStrategy {
+    /*
+     * acquireTokenSilent will first look in the cache before going to the network with the existing refresh token.
+     * The refresh token will be renewed if it is expired.
+     * (existing default behavior)
+     */
+    CacheAndNetwork = 0, // 0 is falsy, is equivalent to not passing in a SilentTokenRetrievalStrategy
+    /*
+     * acquireTokenSilent will only look in the cache, and will not go to the network with the existing refresh token.
+     * The refresh token will not be renewed if it is expired.
```

Comment:
For MSAL libraries, RT expiration and other details are opaque. It is an implementation detail of AAD. I don't think we should document anything related expiration of RT.

Reply:
We attempt to use the RT and if the server tells us it is expired or invalid we attempt to silently renew it using a hidden iframe. Though this brings up an interesting perspective which is that these options are only relevant to msal-browser's implementation of `acquireTokenSilent` - perhaps this enum should live in msal-browser rather than common

Reply:
The enum IS being in msal-common thought. In lib/msal-common/src/client/SilentFlowClient.ts

Reply:
The new logic you've added there can be moved up in the stack to be executed in msal-browser. If we don't envision this enum being used in msal-node (which I don't) then that is likely the right move.